modules/security-context-constraints-about.adoc

```diff
@@ -52,8 +52,7 @@ effectively root on the cluster and must be trusted accordingly.
 
 [IMPORTANT]
 ====
-Do not modify the default SCCs. Customizing the default SCCs can lead to issues
-when {product-title} is upgraded.
+Do not modify the default SCCs. Customizing the default SCCs can lead to issues when some of the platform pods deploy or {product-title} is upgraded. During upgrades between some versions of {product-title}, the values of the default SCCs are reset to the default values, which discards all customizations to those SCCs.
```

Comment:
_During upgrades between some versions of {product-title},_ 
This bugs me for some reason. I read it as a certain category of upgrades resets the SSCs, like all Y-streams, but not Z-stream or something. 
Maybe: _During upgrades of {product-title}, the values of the default SCCs can be reset..._

Feel free to ignore if you think this too fine a point.